dialogflow/cx/apiv3/agents_client.go

```diff
@@ -25,7 +25,6 @@ import (
 
 	"cloud.google.com/go/longrunning"
 	lroauto "cloud.google.com/go/longrunning/autogen"
-	structpb "github.com/golang/protobuf/ptypes/struct"
```

Comment:
Is `dialogflow/cx/apiv3` a beta client?

Reply:
no, This must be remnants of a recent generator upgrade. If you look at the files edited in this folder they are just using the new proto package that was previously aliased. I think we have seen this exact change surface in other PRs recently too. I am not sure how this got into the changeset, but it is safe.

Reply:
Gotcha, thanks!